src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ToolboxItemFilterAttribute.cs

```diff
@@ -69,49 +67,27 @@ public ToolboxItemFilterAttribute(string filterString, ToolboxItemFilterType fil
         {
             if (filterString == null) filterString = String.Empty;
```

Comment:
Instead of doing
```cs
if (filterString == null) filterString = String.Empty
```
we could simplify this to one line when initializing `FilterString` : 

```cs
FilterString = filterString ?? string.Empty;
```